README.md

```diff
@@ -1,55 +1,22 @@
 # schwifty-migration
 
-#### An [Objection.js](https://github.com/Vincit/objection.js) model migration tool for use during development
+#### An [Objection.js](https://vincit.github.io/objection.js) model migration tool for use during development
 
 [![Build Status](https://travis-ci.org/BigRoomStudios/schwifty-migration.svg?branch=master)](https://travis-ci.org/BigRoomStudios/schwifty-migration) [![Coverage Status](https://coveralls.io/repos/github/BigRoomStudios/schwifty-migration/badge.svg?branch=master)](https://coveralls.io/github/BigRoomStudios/schwifty-migration?branch=master) [![Security Status](https://nodesecurity.io/orgs/big-room-studios/projects/3f03e446-4689-49b3-9d82-ab3070eea553/badge)](https://nodesecurity.io/orgs/big-room-studios/projects/3f03e446-4689-49b3-9d82-ab3070eea553)
 
-## Getting Started
-- Run `npm install`
-- Install the right version of 'schwifty'
-```
-npm install git+https://github.com/BigRoomStudios/schwifty.git#add-schwifty-migration
-```
-- Install 'schwifty-migration'
-```
-npm install git+https://github.com/BigRoomStudios/schwifty-migration.git
-```
----
-
-- Create an npm 'migrate' script so that schwifty's `migrationsMode` plugin option is set to either 'create' or 'alter'.
+## Usage with [`schwifty`](https://github.com/BigRoomStudios/schwifty) by means of [`hpal`](https://github.com/devinivy/hpal)
```

Comment:
Usage-wise, the most important thing for a user to know is that,

1. it's used by schwifty to implement the `hpal run schwifty:migrate` command (although, it sounds like we migth want to rename that command).
2. In order to use that command, all you have to do is `npm install --save-dev schwifty-migration` in your project that uses schwifty.

I think the command documentation should live in schwifty's docs.

Reply:
What should we rename the command to?
I was trying to get rid of confusion with the plural of `migration`
schwifty-migration is singular, and I want to keep that consistent. (migrations VS migration)

In thinking about the CLI I thought gen-migration-file was too verbose

Howbout `gen-migration`?

Reply:
Might be worth pinging Bri and Matt.  In symfony, for example, you would run `doctrine:migrate:diff`

Reply:
Worth keeping in mind we may add other migrations commands to schwifty, e.g. `schwifty:migrations:rollback`.

Reply:
Glad symfony calls it a `diff` too! I'm all up for mentioning the diffing that this module does

Reply:
Would `schwifty:migrations:rollback` have different functionality from knex's CLI rollback?

Reply:
Yes, but only in the case that there are multiple plugins with migrations.